docs/EN_US/HPCCClientTools/CT_Mods/ECLCC.xml

```diff
@@ -185,8 +185,10 @@
               <row>
                 <entry>-main <emphasis>&lt;ref&gt;</emphasis></entry>
```

Comment:
should this be --main (two hyphens)?

Reply:
that is the way it appears in the existing doc, it was not changed